docs/source/repository_structure.rst

```diff
@@ -0,0 +1,141 @@
+Structure your repository
+=========================
+
+To host and share your dataset, you can create a dataset repository on the Hugging Face Dataset Hub and upload your data files.
+
+This guide aims to explain what structure is expected when you upload a dataset.
+A dataset with a supported structure can be loaded automatically using ``load_dataset``, and also has a nice preview on its dataset page on the Hub.
```

Comment:
```suggestion
A dataset with a supported structure can be loaded automatically with ``load_dataset``, and it will have a preview on its dataset page on the Hub.
```